packages/react-core/src/components/DescriptionList/__tests__/DescriptionList.test.tsx

```diff
@@ -1,130 +1,110 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
```

Comment:
Could we add additional tests to check that the `style` object is being applied correctly when the applicable props are possed? isHorizontal + horizontalTermWidthModifier, termWidth, and isAutoFit + autoFitMinModifier